pkg/client/s3client.go

```diff
@@ -299,13 +373,15 @@ func (c *S3Client) UploadObject(fileName, objectName, bucketName string) error {
 		Body:   reader,
 	}
 
-	// Perform an upload.
+	// Perform an upload
 	startTime := time.Now()
 	result, err := uploader.Upload(upParams)
 	if err != nil {
-		return err
+		return fmt.Errorf("upload failed: %v", err)
 	}
-	fmt.Println()
+
+	progressTracker.progress.Wait()
+
 	klog.Infof("Upload completed successfully in %s seconds to location %s", time.Since(startTime).Round(time.Second), result.Location)
```

Comment:
I1126 21:33:18.874906   97504 s3client.go:384] Upload completed successfully in 1m44s `seconds` to location https://s3.jp-tok.xxxxx.xxxx/xxxx/xxxx/xx/xx/xx/xx/
```suggestion
	klog.Infof("Upload completed successfully in %s to location %s", time.Since(startTime).Round(time.Second), result.Location)
```

Maybe we need to remove `seconds`.

Reply:
Thanks for the patch @kishen-v 
Applied suggested changes